atd-vzd/migrations/default/1693516393057_alter_table_public_atd_txdot_crashes_drop_column_micromobility_device_flag/down.sql

```diff
@@ -0,0 +1 @@
+alter table "public"."atd_txdot_crashes" add "column micromobility_device_flag" character varying(1) DEFAULT 'N'::character varying NOT NULL,
```

Comment:
I tested running this down migration and ran into an error with some syntax in here. `column` from `add column` made its way into the column name, and there is a comma at the end of the statement instead of a semicolon. I tested it with those changes, and it works perfectly!

Reply:
thanks for catching those typos! i fixed them and confirmed the migration worked